lib/stores/GCSDataStore.js

```diff
@@ -131,9 +133,11 @@ class GCSDataStore extends DataStore {
             return new Promise((resolve, reject) => {
                 const file = this.bucket.file(file_id);
 
+                const parsedMetadata = this._parseMetadataString(data.upload_metadata);
                 const options = {
                     offset,
                     metadata: {
+                        contentType: parsedMetadata.type.decoded,
```

Comment:
Again, a check to see if `parsedMetadata.type` actually exists might be necessary.

Reply:
you're right 👍 will add it